scipy/interpolate/fitpack2.py

```diff
@@ -120,12 +120,38 @@ class UnivariateSpline(object):
 
     See Also
     --------
-    InterpolatedUnivariateSpline : Subclass with smoothing forced to 0
-    LSQUnivariateSpline : Subclass in which knots are user-selected instead of
-        being set by smoothing condition
-    splrep : An older, non object-oriented wrapping of FITPACK
-    splev, sproot, splint, spalde
-    BivariateSpline : A similar class for two-dimensional spline interpolation
+    BivariateSpline :
+        a base class for bivariate splines.
+    SmoothBivariateSpline :
+        a smooth bivariate spline through the given points
```

Comment:
I think the term is smooth*ing* spline (all of them are smooth, but this one does smoothing of the noise in the input data)

Reply:
Agree. I fixed it.